core/tests/test_network.py

```diff
@@ -0,0 +1,38 @@
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.docker_client import DockerClient
+from testcontainers.core.network import Network
+
+NGINX_ALPINE_SLIM_IMAGE = "nginx:alpine-slim"
```

Comment:
Can you pin this dpenendency to a specific, versioned tag? We are currently working on making our tests more reproducible, and specific version tags for images used in tests are a part of this.

Reply:
fixed in 9db1080c416c26f9cb4f542befe19dd17958ffa1